frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -139,6 +140,20 @@ export const createDeploymentConfig = (
               image: `${name}:latest`,
               ports,
               env,
+              resources: {
+                ...((cpu.limit || memory.limit) && {
+                  limits: {
+                    ...(cpu.limit && { cpu: `${cpu.limit}${cpu.limitUnit}` }),
```

Comment:
have you validated with various units or we need some conversion before applying?

Reply:
I've tried different combinations. It works.